.travis.yml

```diff
@@ -44,26 +54,30 @@ before_install:
   - do_on(){ if [ "$TRAVIS_OS_NAME" = "$1" ]; then shift; "$@" ; fi; }
   - on_host(){ if [ -z "$DOCKER_IMG_JM" ]; then "$@" ; fi; }
   - should_run_dockers(){ if [ "$TRAVIS_EVENT_TYPE" = cron ] || [ -n "$TRAVIS_TAG" ] || echo "${TRAVIS_COMMIT_MESSAGE[@]}" | grep -q "TRAVIS_RUN_DOCKERS"; then return 0; else return 1; fi; }
-  - on_docker(){ if [ -n "$DOCKER_IMG_JM" ] && should_run_dockers; then "$@" ; fi; }
+  - on_docker(){ if [ -n "$DOCKER_IMG_JM" ]; then "$@" ; fi; }
   - do_on osx pip install virtualenv
 cache:
   directories:
    - $HOME/downloads
    - $HOME/.cache/pip
    - $HOME/Library/Caches/pip
+   - $HOME/Library/Caches/Homebrew
+before_cache:
+  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then brew cleanup; fi
 install:
   - mkdir -p "$HOME/downloads"
   - mkdir -p "$TRAVIS_BUILD_DIR/deps/cache/"
   - find "$HOME/downloads" -type f -exec cp -v {} "$TRAVIS_BUILD_DIR/deps/cache/" \;
-  - on_host ./install.sh --develop --python=python3 --with-qt
+  - on_host do_on linux ./install.sh --develop --python=python3 --with-qt
```

Comment:
Passing `--develop` to `install.sh` is now useless.

Reply:
It is but leaving for legacy reasons. Maybe we decide to not use `-e` when installing the jm packages.